drivers/bridge/errors.go

```diff
@@ -178,6 +178,17 @@ func (inie InvalidNetworkIDError) Error() string {
 // NotFound denotes the type of this error
 func (inie InvalidNetworkIDError) NotFound() {}
 
+// NetworkNotFoundError is returned when the no network
```

Comment:
We are moving away from this internal error types, in favor of the error interface types.
Sorry, we have not removed them from the code yet. Please look at open issue #269